
第1条测试用例19-05-25 18:12-ERROR-Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 80, in go_on_run
    res=self.run_method.run_main(method,url,data,header,params=data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 10, in post_main
    res=requests.post(url=url,data=data,headers=header)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?
Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 80, in go_on_run
    res=self.run_method.run_main(method,url,data,header,params=data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 10, in post_main
    res=requests.post(url=url,data=data,headers=header)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?

第2条测试用例19-05-25 18:12-ERROR-Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 80, in go_on_run
    res=self.run_method.run_main(method,url,data,header,params=data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 34, in run_main
    res=self.get_main(url,params,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 23, in get_main
    res=requests.get(url=url,params=params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?
Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 80, in go_on_run
    res=self.run_method.run_main(method,url,data,header,params=data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 34, in run_main
    res=self.get_main(url,params,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 23, in get_main
    res=requests.get(url=url,params=params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?
Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 80, in go_on_run
    res=self.run_method.run_main(method,url,data,header,params=data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 34, in run_main
    res=self.get_main(url,params,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 23, in get_main
    res=requests.get(url=url,params=params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?

第3条测试用例19-05-25 18:12-ERROR-Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 80, in go_on_run
    res=self.run_method.run_main(method,url,data,header,params=data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 12, in post_main
    res=requests.post(url=url,data=data)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?
Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 80, in go_on_run
    res=self.run_method.run_main(method,url,data,header,params=data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 12, in post_main
    res=requests.post(url=url,data=data)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?
Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 80, in go_on_run
    res=self.run_method.run_main(method,url,data,header,params=data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 12, in post_main
    res=requests.post(url=url,data=data)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?
Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 80, in go_on_run
    res=self.run_method.run_main(method,url,data,header,params=data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 12, in post_main
    res=requests.post(url=url,data=data)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?

第4条测试用例19-05-25 18:12-ERROR-Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 61, in go_on_run
    depend_response_data=self.depend_data.get_value_for_key(i)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\data\dependent_data.py", line 41, in get_value_for_key
    reponse_data=self.run_dependent()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\data\dependent_data.py", line 30, in run_dependent
    res=self.method.run_main(method,url,request_data,header,params=request_data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 12, in post_main
    res=requests.post(url=url,data=data)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?
Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 61, in go_on_run
    depend_response_data=self.depend_data.get_value_for_key(i)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\data\dependent_data.py", line 41, in get_value_for_key
    reponse_data=self.run_dependent()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\data\dependent_data.py", line 30, in run_dependent
    res=self.method.run_main(method,url,request_data,header,params=request_data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 12, in post_main
    res=requests.post(url=url,data=data)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?
Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 61, in go_on_run
    depend_response_data=self.depend_data.get_value_for_key(i)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\data\dependent_data.py", line 41, in get_value_for_key
    reponse_data=self.run_dependent()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\data\dependent_data.py", line 30, in run_dependent
    res=self.method.run_main(method,url,request_data,header,params=request_data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 12, in post_main
    res=requests.post(url=url,data=data)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?
Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 61, in go_on_run
    depend_response_data=self.depend_data.get_value_for_key(i)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\data\dependent_data.py", line 41, in get_value_for_key
    reponse_data=self.run_dependent()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\data\dependent_data.py", line 30, in run_dependent
    res=self.method.run_main(method,url,request_data,header,params=request_data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 12, in post_main
    res=requests.post(url=url,data=data)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?
Invalid URL '': No schema supplied. Perhaps you meant http://?
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 61, in go_on_run
    depend_response_data=self.depend_data.get_value_for_key(i)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\data\dependent_data.py", line 41, in get_value_for_key
    reponse_data=self.run_dependent()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\data\dependent_data.py", line 30, in run_dependent
    res=self.method.run_main(method,url,request_data,header,params=request_data)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 31, in run_main
    res=self.post_main(url,data,header)
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\method_run.py", line 12, in post_main
    res=requests.post(url=url,data=data)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\sessions.py", line 441, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 309, in prepare
    self.prepare_url(url, params)
  File "C:\Program Files\Anaconda3\lib\site-packages\requests\models.py", line 383, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL '': No schema supplied. Perhaps you meant http://?

第5条测试用例19-05-25 18:12-ERROR-string indices must be integers
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 71, in go_on_run
    op_header.write_cookie()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 28, in write_cookie
    cookie=requests.utils.dict_from_cookiejar(self.get_cookie())
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 22, in get_cookie
    url=self.get_response_url()+'&callback=jQuery21008240514814031887_1508666806688&_=1508666806689'
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 17, in get_response_url
    url=self.response['data']['url'][0]
TypeError: string indices must be integers
string indices must be integers
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 71, in go_on_run
    op_header.write_cookie()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 28, in write_cookie
    cookie=requests.utils.dict_from_cookiejar(self.get_cookie())
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 22, in get_cookie
    url=self.get_response_url()+'&callback=jQuery21008240514814031887_1508666806688&_=1508666806689'
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 17, in get_response_url
    url=self.response['data']['url'][0]
TypeError: string indices must be integers
string indices must be integers
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 71, in go_on_run
    op_header.write_cookie()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 28, in write_cookie
    cookie=requests.utils.dict_from_cookiejar(self.get_cookie())
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 22, in get_cookie
    url=self.get_response_url()+'&callback=jQuery21008240514814031887_1508666806688&_=1508666806689'
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 17, in get_response_url
    url=self.response['data']['url'][0]
TypeError: string indices must be integers
string indices must be integers
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 71, in go_on_run
    op_header.write_cookie()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 28, in write_cookie
    cookie=requests.utils.dict_from_cookiejar(self.get_cookie())
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 22, in get_cookie
    url=self.get_response_url()+'&callback=jQuery21008240514814031887_1508666806688&_=1508666806689'
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 17, in get_response_url
    url=self.response['data']['url'][0]
TypeError: string indices must be integers
string indices must be integers
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 71, in go_on_run
    op_header.write_cookie()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 28, in write_cookie
    cookie=requests.utils.dict_from_cookiejar(self.get_cookie())
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 22, in get_cookie
    url=self.get_response_url()+'&callback=jQuery21008240514814031887_1508666806688&_=1508666806689'
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 17, in get_response_url
    url=self.response['data']['url'][0]
TypeError: string indices must be integers
string indices must be integers
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 71, in go_on_run
    op_header.write_cookie()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 28, in write_cookie
    cookie=requests.utils.dict_from_cookiejar(self.get_cookie())
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 22, in get_cookie
    url=self.get_response_url()+'&callback=jQuery21008240514814031887_1508666806688&_=1508666806689'
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_header.py", line 17, in get_response_url
    url=self.response['data']['url'][0]
TypeError: string indices must be integers

第6条测试用例19-05-25 18:12-ERROR-Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 74, in go_on_run
    op_json=OperationJson('../dataconfig/cookie.json')
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 8, in __init__
    self.data=self.read_data()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 14, in read_data
    data=json.load(f)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 74, in go_on_run
    op_json=OperationJson('../dataconfig/cookie.json')
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 8, in __init__
    self.data=self.read_data()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 14, in read_data
    data=json.load(f)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 74, in go_on_run
    op_json=OperationJson('../dataconfig/cookie.json')
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 8, in __init__
    self.data=self.read_data()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 14, in read_data
    data=json.load(f)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 74, in go_on_run
    op_json=OperationJson('../dataconfig/cookie.json')
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 8, in __init__
    self.data=self.read_data()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 14, in read_data
    data=json.load(f)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 74, in go_on_run
    op_json=OperationJson('../dataconfig/cookie.json')
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 8, in __init__
    self.data=self.read_data()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 14, in read_data
    data=json.load(f)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 74, in go_on_run
    op_json=OperationJson('../dataconfig/cookie.json')
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 8, in __init__
    self.data=self.read_data()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 14, in read_data
    data=json.load(f)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "F:/github/Excel-requests-unittest-STMP-HTMLTestReport/main/run_test.py", line 74, in go_on_run
    op_json=OperationJson('../dataconfig/cookie.json')
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 8, in __init__
    self.data=self.read_data()
  File "F:\github\Excel-requests-unittest-STMP-HTMLTestReport\util\operation_json.py", line 14, in read_data
    data=json.load(f)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "C:\Program Files\Anaconda3\lib\json\__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Program Files\Anaconda3\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
